helm/values.yaml

```diff
@@ -32,12 +46,37 @@ env:
   logPrettyPrintEnabled: false
   responseCompressionEnabled: true
   requestPayloadLimit: 1mb
-  tracing:
-    enabled: false
-    url: http://localhost:55681/v1/trace
-  metrics:
-    enabled: false
-    url: http://localhost:55681/v1/metrics
+  application:
+    arraySeparator: ','
+    createPolygonPartsTablesStoredProcedure: 'create_polygon_parts_tables'
+    updatePolygonPartsTablesStoredProcedure: 'update_polygon_parts'
+    entities:
+      parts:
+        namePrefix: ''
+        nameSuffix: '_parts'
+      polygonParts:
+        namePrefix: ''
+        nameSuffix: ''
+
+# external
+db:
```

Comment:
We need those db values to be empty strings because were using the tpl values which define this scope as stronger than the global values

Reply:
set `host`, `username`, `password`, `name` to empty string